persistence/persistence-mybatis/src/test/resources/testSql.sql

```diff
@@ -346,10 +346,10 @@ INSERT INTO clinical_attribute_meta (ATTR_ID,DISPLAY_NAME,DESCRIPTION,DATATYPE,P
-INSERT INTO structural_variant (SAMPLE_ID,BREAKPOINT_TYPE,ANNOTATION,COMMENTS,CONFIDENCE_CLASS,CONNECTION_TYPE,EVENT_INFO,MAPQ,NORMAL_READ_COUNT,NORMAL_VARIANT_COUNT,PAIRED_END_READ_SUPPORT,SITE1_CHROM,SITE1_DESC,SITE1_ENTREZ_GENE_ID,SITE1_POS,SITE2_CHROM,SITE2_DESC,SITE2_ENTREZ_GENE_ID,SITE2_POS,SPLIT_READ_SUPPORT,SV_CLASS_NAME,SV_DESC,SV_LENGTH,TUMOR_READ_COUNT,TUMOR_VARIANT_COUNT,VARIANT_STATUS_NAME,GENETIC_PROFILE_ID) VALUES(2,'PRECISE','n/a','n/a','AUTO_OK','3to5','Protein fusion: mid-exon (ERBB2-GRB7)',0,7062,0,97,'17','Exon 25 of ERBB2(+)',2064,37883138,'17','Intron of GRB7(+): 56bp before exon 10',2886,37901416,71,'DELETION','n/a',18278,9849,60,'NEW_VARIANT',7);
-INSERT INTO structural_variant (SAMPLE_ID,BREAKPOINT_TYPE,ANNOTATION,COMMENTS,CONFIDENCE_CLASS,CONNECTION_TYPE,EVENT_INFO,MAPQ,NORMAL_READ_COUNT,NORMAL_VARIANT_COUNT,PAIRED_END_READ_SUPPORT,SITE1_CHROM,SITE1_DESC,SITE1_ENTREZ_GENE_ID,SITE1_POS,SITE2_CHROM,SITE2_DESC,SITE2_ENTREZ_GENE_ID,SITE2_POS,SPLIT_READ_SUPPORT,SV_CLASS_NAME,SV_DESC,SV_LENGTH,TUMOR_READ_COUNT,TUMOR_VARIANT_COUNT,VARIANT_STATUS_NAME,GENETIC_PROFILE_ID) VALUES(3,'PRECISE','n/a','ERBB2 (NM_004448) rearrangement','MANUAL_OK','3to5','Deletion of 1 exon: in frame',0,7212,0,31,'17','Intron of ERBB2(+): 46bp after exon 15',2064,37873779,'17','Intron of ERBB2(+): 501bp before exon 17',2064,37879041,10,'DELETION','n/a',5262,3101,7,'NEW_VARIANT',7);
-INSERT INTO structural_variant (SAMPLE_ID,BREAKPOINT_TYPE,ANNOTATION,COMMENTS,CONFIDENCE_CLASS,CONNECTION_TYPE,EVENT_INFO,MAPQ,NORMAL_READ_COUNT,NORMAL_VARIANT_COUNT,PAIRED_END_READ_SUPPORT,SITE1_CHROM,SITE1_DESC,SITE1_ENTREZ_GENE_ID,SITE1_POS,SITE2_CHROM,SITE2_DESC,SITE2_ENTREZ_GENE_ID,SITE2_POS,SPLIT_READ_SUPPORT,SV_CLASS_NAME,SV_DESC,SV_LENGTH,TUMOR_READ_COUNT,TUMOR_VARIANT_COUNT,VARIANT_STATUS_NAME,GENETIC_PROFILE_ID) VALUES(4,'PRECISE','n/a','n/a','MANUAL_OK','3to5','Deletion of 1 exon',0,5328,0,52,'17','Intron of ERBB2(+): 196bp after exon 15',2064,37873929,'17','Intron of ERBB2(+): 442bp after exon 16',2064,37876529,33,'DELETION','n/a',2600,19320,13,'NEW_VARIANT',7);
+--INSERT INTO structural_variant (SAMPLE_ID,BREAKPOINT_TYPE,ANNOTATION,COMMENTS,CONFIDENCE_CLASS,CONNECTION_TYPE,EVENT_INFO,MAPQ,NORMAL_READ_COUNT,NORMAL_VARIANT_COUNT,PAIRED_END_READ_SUPPORT,SITE1_CHROM,SITE1_DESC,SITE1_ENTREZ_GENE_ID,SITE1_POS,SITE2_CHROM,SITE2_DESC,SITE2_ENTREZ_GENE_ID,SITE2_POS,SPLIT_READ_SUPPORT,SV_CLASS_NAME,SV_DESC,SV_LENGTH,TUMOR_READ_COUNT,TUMOR_VARIANT_COUNT,VARIANT_STATUS_NAME,GENETIC_PROFILE_ID) VALUES(1,'PRECISE','n/a','n/a','AUTO_OK','3to5','Transcript fusion (ERBB2-GRB7)',0,7181,0,11,'17','Intron of ERBB2(+): 51bp after exon 26',2064,37883851,'17','5-UTR of GRB7(+): 1Kb before coding start',2886,37897379,31,'DELETION','n/a',13528,4389,6,'NEW_VARIANT',7);
```

Comment:
please put these back, but you can limit it to the FUSION use cases for now. Other SVs should be added/provided later by @n1zea144 and others looking to support SVs other than FUSIONs. These will be likely added as new INSERT statements rather than extra columns in an existing INSERT here.

Reply:
structural_variant INSERT statements are now included in `testSql.sql`. They only describe fusion events.